internal/controller/isbservicerollout/isbservicerollout_controller.go

```diff
@@ -986,6 +986,10 @@ func (r *ISBServiceRolloutReconciler) listAndDeleteChildISBServices(ctx context.
 	isbServiceList, err := kubernetes.ListLiveResource(ctx, common.NumaflowAPIGroup, common.NumaflowAPIVersion, numaflowv1.ISBGroupVersionResource.Resource,
 		isbServiceRollout.Namespace, fmt.Sprintf("%s=%s", common.LabelKeyParentRollout, isbServiceRollout.Name), "")
 	if err != nil {
+		if !apierrors.IsNotFound(err) {
```

Comment:
am I mistaken? I would think this would be opposite: `if apierrors.IsNotFound(err)`

Reply:
Ahh, Right. Let me fix it